src/digraphs.c

```diff
@@ -854,6 +854,85 @@ static Obj FuncDIGRAPH_TOPO_SORT(Obj self, Obj adj) {
   return out;
 }
 
+// TODO(*) use generic DFS, when we have one.
+
```

Comment:
Please also add some explanation of what you are doing here, can be a sentence or two.